qualtran/linalg/lcu_util.py

```diff
@@ -158,7 +158,7 @@ def sub_bit_prec_from_epsilon(n: SymbolicInt, epsilon: SymbolicFloat) -> Symboli
 
 
 def sub_bit_prec_from_epsilon(n: SymbolicInt, epsilon: SymbolicFloat) -> SymbolicInt:
-    ret = ceil(-log2(epsilon * n))
+    ret = ceil(log2(1 / (epsilon * n)))
```

Comment:
Is this just more aesthetically pleasing?

Reply:
Yes